src/quo/components/wallet/progress_bar/view.cljs

```diff
@@ -5,9 +5,11 @@
     [react-native.core :as rn]))
 
 (defn- view-internal
-  [props]
+  [{:keys [full-width?] :as props}]
```

Comment:
are these ever not full width? 🤔

Reply:
@J-Son89 As far as I understood, `progress-bar` is actually a small box, one of the many we use in `transaction-progress`. (not very lucky naming) 
But in some cases we need it to be full-width, like when network is `optimism` or `arbitrum` in quo component preview.
@mmilad75, did I get it correctly? If so, I'd suggest to add `full-width?` prop to `progress-bar` component preview.

Reply:
@vkjr You're right. I will add it 👍 . 
cc @J-Son89.

Reply:
Added